pkg/image/validation/registry.go

```diff
@@ -0,0 +1,97 @@
+package validation
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/suse-edge/edge-image-builder/pkg/combustion"
+	"github.com/suse-edge/edge-image-builder/pkg/image"
+)
+
+const (
+	registryComponent = "Artifact Registry"
+)
+
+func validateEmbeddedArtefactRegistry(ctx *image.Context) []FailedValidation {
```

Comment:
This is a nit but `Artifact` is only spelled with an "e" once in the whole file.

Reply:
I know, I was trying to keep consistent with the other places in the code that use Artefact, forgetting it was part of the definition. I'll revert these for consistency within the file.